test/optimization/test_grover_optimizer.py

```diff
@@ -113,7 +115,25 @@ def test_qubo_gas_int_paper_example(self):
 
         # Get the optimum key and value.
         n_iter = 10
-        gmf = GroverOptimizer(6, num_iterations=n_iter, quantum_instance=self.q_instance)
+        gmf = GroverOptimizer(6, num_iterations=n_iter, quantum_instance=self.sv_simulator)
+        results = gmf.solve(op)
+        self.validate_results(op, results)
+
+    def test_qubo_gas_int_paper_example_qasm(self):
```

Comment:
instead of copying the test you could just run the above for two different simulators using the `@ddt` and `@data` decorators, as
```python
@ddt
class TestGroverOptimizer(QiskitAquaTestCase):
   ...
   @data('sv', 'qasm')
   def test_qubo_gas_int_paper_example(self, simulator):
        ...
        q_instance = self.sv_simulator if simulator == 'sv' else self.qasm_simulator
        ...

Reply:
That's cooler. I will change it as you mentioned.